style.css

```diff
@@ -0,0 +1,1034 @@
+  --badge-danger-background-color: #dc3545;
+  --badge-warning-background-color: #ffc107;
+  --badge-info-background-color: #0dcaf0;
+  --badge-light-background-color: #f8f9fa;
+  --badge-dark-background-color: black; */
+
+  /* --primary-color: #0d6efd; */
+  --nav-color:orange;
+  --primary-color: #1629d4;
+  --secondary-color: #6c757d;
+  --success-color: rgb(9, 129, 9);
+  
+  --danger-color: rgb(240, 20, 20);
+  --warning-color: #ffc107;
+  --info-color: #0dcaf0;
+  --light-color: #f8f9fa;
+  --dark-color: black;
+}
+
+.header{
+  background-color: var(--alert-secondary-font-color);
+  color: var(--nav-color);
+  padding: 5px;
+  text-align: center;
+}
+
+/* alerts */
+
+.alert {
+  border: 1px solid transparent;
+  border-radius: 5px;
+  padding: 1rem;
+  margin: 1rem 0;
+  position: relative;
+}
+
+.alert-icon {
+  font-size: 150%;
+  vertical-align: middle;
+  margin-right: 0.5rem;
+}
+
+.alert-close {
+  position: absolute;
+  top: 0.25rem;
+  right: 0.25rem;
+  cursor: pointer;
+}
+
+.alert-primary {
+  background-color: var(--alert-primary-background-color);
+  color: var(--alert-primary-font-color);
+}
+
+.alert-secondary {
+  background-color: var(--alert-secondary-background-color);
+  color: var(--alert-secondary-font-color);
+}
+
+.alert-success {
+  background-color: var(--alert-success-background-color);
+  color: var(--alert-success-font-color);
+}
+
+.alert-danger {
+  background-color: var(--alert-danger-background-color);
+  color: var(--alert-danger-font-color);
+}
+
+.alert-warning {
+  background-color: var(--alert-warning-background-color);
+  color: var(--alert-warning-font-color);
+}
+
+.alert-info {
+  background-color: var(--alert-info-background-color);
+  color: var(--alert-info-font-color);
+}
+
+.alert-light {
+  background-color: var(--alert-light-background-color);
+  color: var(--alert-light-font-color);
+}
+
+.alert-dark {
+  background-color: var(--alert-dark-background-color);
+  color: var(--alert-dark-font-color);
+}
+
+/* Badges */
+
+.badge {
+  border: 1px solid transparent;
+  border-radius: 0.25rem;
+  color: var(--text-color);
+  /* margin-top: 1rem; */
+  padding: 0.25rem 0.5rem;
+  font-size: 0.75rem;
+  font-weight: 700;
+}
+
+.ic-bg {
+  font-size: 200%;
+}
+
+.bg-icon {
+  position: relative;
+  top: -1.5rem;
+  left: -0.8rem;
+}
+
+.rounded-pill {
+  border-radius: 1rem;
+}
+
+.bg-primary {
+  background-color: var(--primary-color);
+}
+
+.bg-secondary {
+  background-color: var(--secondary-color);
+}
+
+.bg-success {
+  background-color: var(--success-color);
+}
+
+.bg-danger {
+  background-color: var(--danger-color);
+}
+
+.bg-warning {
+  background-color: var(--warning-color);
+}
+
+.bg-info {
+  background-color: var(--info-color);
+}
+
+.bg-light {
+  background-color: var(--light-color);
+  color: var(--text-dark-color);
+}
+
+.bg-dark {
+  background-color: var(--dark-color);
+}
+
+/* Buttons */
+
+.btn {
+  outline: none;
+  margin: 0.25rem;
```

Comment:
you should not do ouline: none, 
for accessibilty it is needed.

Reply:
refactored roshan thanks!!